lib/tasks/admin.rake

```diff
@@ -0,0 +1,29 @@
+require 'io/console'
+
+namespace :admin do
+  desc 'Creates new admin user'
+  task :new => :environment do
+    user_params = {}
+    puts 'Enter the following details to make a new admin'
+    print 'First Name : ' 
+    user_params[:first_name] = STDIN.gets.chomp
+    print 'Last Name : '
+    user_params[:last_name] = STDIN.gets.chomp
+    print 'Email : '
+    user_params[:email] = STDIN.gets.chomp
+    print 'Password : '
+    user_params[:password] = STDIN.noecho(&:gets).chomp
+    puts
+    print 'Confirm Password : '
+    user_params[:password_confirmation] =  STDIN.noecho(&:gets).chomp
+    puts
+    user_params[:verified_at] = Time.now
+
+    admin = User.new(user_params)
```

Comment:
Why haven't you added admin role here?
Are you creating an admin or a normal user here ?